### PR TITLE
Added Repo Info

### PR DESCRIPTION
**Pull request by VFXGamer**
Repo Name PhishBuster

### CODE REVIEW

**Comment by TheLastProject** [comment]
Please add some [Github Issues](https://github.com/VFXGamer/PhishBuster/issues) so people know what to contribute to.

Also, this repo is missing the hacktoberfest repo topic.

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
 
 ## Python
 - [F-Droid Server](https://gitlab.com/fdroid/fdroidserver/) - Tools for managing builds, indexes, updates, and deployments for F-Droid repositories.
+- [PhishBuster](https://github.com/VFXGamer/PhishBuster) - A Phishing Site Detection Tool (Web Based)
```
Comment (TheLastProject) [inline README.md:23]: Please add some Github Issues so people know what to contribute to.

Also, this repo is missing the hacktoberfest repo topic.

**Review by TheLastProject — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (VFXGamer) [reply README.md:23]: What do you mean by "missing the hacktoberfest repo topic" this is my first time as a maintainer.

Comment (TheLastProject) [reply README.md:23]: Maybe I misread earlier, I'm seeing it now: 
image

So approving this :)

**Pull request merged**